generate_R_table.py

```diff
@@ -1,3 +1,4 @@
+#!/Users/antonmalko/anaconda/bin/ python
```

Comment:
Haha, I'm a controlling bastard, so yes, I've set up notifications ;)
I recently got a bug report from someone in Brazil, so I think we should definitely try to keep very specific configurations to a minimum.